src/Apps/Artwork/Components/ArtworkChatBubble.tsx

```diff
@@ -43,17 +42,16 @@ const ArtworkZendesk: FC<ArtworkZendeskProps> = ({ artwork }) => {
   return getENV("SALESFORCE_CHAT_ENABLED") ? (
```

Comment:
That's the thing, in Force we never had this feature with Zendesk, we only introduced it because of handling or SF or Zendesk exclusively that's why I thought we'd get rid of it in Force to be as it was before. I'm going to unblock the PR since still an open question.

Reply:
Yeah, I thought the same. I just don't have clarity over pricing details and sandboxes usage (and if those differ from Zendesk to SF).